### ISSUE TITLE
Banned artist / запрещённые публикации художников 

### ISSUE THREAD

**Issue by iDolmatov** [stalkerg/anime-pictures#119]
Предлагаю сделать для тегов художников (artist) дополнительное обозначение, когда художник просил запретить распространение его работ.
Описание с [https://danbooru.donmai.us/wiki_pages/banned_artist](danbooru).
Это полезно, чтобы пользователь не тратил время на заполнение тегов и знал, что рисунки данного художника подлежат цензуре в плане распространения.
<><><>
Реализовать можно так:
Добавленные тег к новой картинки (new) зачёркивается, а вместо количества пишется ban. При нажатии на тег дополнительно к описанию тега добавляется универсальное описание, что по запросу художника распространение рисунков ограничено.
_
При условии, если нет полного запрета на рисунки художника:
После одобрения к публикации отображение тега возвращается к привычному варианту с сохранением примечания на странице тегов. Если нужно всегда обозначать наличие бана по тегу, то можно обозначать его фон или поле с количеством картинок другим цветом.

**Comment by stalkerg** [comment]
Да, что то подобное нужно. Хотя мне кажется важнее сделать другую часть этого пазла - форму для авторов.

**Comment by MiTereKun** [comment]
а может если уже пользователь загрузил - то банить, но оставлять в списке картинок и прописывать редерект с картинки на ту же, но на пиксиив (или другую площадку автора). В поле где указывается статус картинки писать название площадки, там образом можно будет найти картинку по русским тегам у нас и выполнять требования автора. Ну заливатору все равно надо давать какое-то уведомление, что автор забанен и у вас будет только редирект.

Хотя такое лучше предварительно обсуждать с самим авторам, которые попросил оставить их интеллектуальную собственность без перезалива
